Resources/Views/leaderboard.leaf

```diff
@@ -72,6 +74,60 @@ LeaderboardUser
             </div> 
         #endfor
         <br>
+        <button class="show-more-durations">Show More</button>
+        <script>
+            $(document).ready(function(){
+                var listWorkouts = $(".total-workout");
+                var listDurations = $(".total-duration");
+                var numToShow = 3;
+                var buttonWorkouts = $(".show-more-workouts");
+                var buttonDurations = $(".show-more-durations");
+                var numInListWorkouts = listWorkouts.length;
+                var numInListDurations = listDurations.length;
+
+                listWorkouts.hide();
+                listDurations.hide();
+
+                if(numInListWorkouts <= 3 ){
+                    buttonWorkouts.hide();
+                }
+                if (numInListWorkouts > numToShow + 1) {
+                    buttonWorkouts.show();
+                }
```

Comment:
It doesn't seem like both these if statements are necessary. Is there a situation where both could be `true`?